app/javascript/application.js

```diff
@@ -0,0 +1,17 @@
+import '@hotwired/turbo-rails';
+import './controllers';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from './redux/store';
+import App from './components/App';
```

Comment:
- [ ] There are some missing package error when starting your application. Kindly check it and sure your application run without any error.
![Screenshot from 2024-02-03 01-05-49](https://github.com/Kanyorok/Hello-React/assets/91142432/3fe42e86-4e97-4aee-bad5-b53cac6fd140)

Reply:
@LionRouge1 could you help me in running ./bin/dev. When I run it i get an issue that it is not recognized command